backend/core/types/src/backend-swagger.ts

```diff
@@ -1625,7 +1625,7 @@ export class MultiselectQuestionsService {
     })
   }
   /**
-   * Get Listings by multiselect question id
+   * Get multiselect question by id
```

Comment:
Why did this change happen here? It also looks like it is now inaccurate

Reply:
Hey @emilyjablonski I'm unsure when that controller's summary changed I changed it to match more closely with what the endpoint does, lemme know what you think